pkg/gadgets/trace/capabilities/types/types.go

```diff
@@ -29,15 +32,29 @@ const (
 type Event struct {
 	eventtypes.Event
 
-	MountNsID uint64 `json:"mountnsid,omitempty"`
-	Pid       uint32 `json:"pid,omitempty"`
-	UID       uint32 `json:"uid,omitempty"`
-	Comm      string `json:"comm,omitempty"`
-	CapName   string `json:"capName,omitempty"`
-	Cap       int    `json:"cap,omitempty"`
-	Audit     int    `json:"audit,omitempty"`
-	InsetID   *bool  `json:"insetid,omitempty"`
-	Verdict   string `json:"verdict,omitempty"`
+	Pid       uint32 `json:"pid,omitempty" column:"pid,width:7,fixed"`
+	Comm      string `json:"comm,omitempty" column:"comm,width:16,fixed"`
+	UID       uint32 `json:"uid,omitempty" column:"uid,width:10,fixed"`
+	Cap       int    `json:"cap,omitempty" column:"cap,width:3,fixed"`
+	CapName   string `json:"capName,omitempty" column:"capName,width:20,ellipsis:start"`
```

Comment:
This will change the column name from `NAME` to `CAPNAME`. Do we all agree with this or should I switch it back?

Reply:
I think `CAPNAME` is more descriptive.